reaktive/src/jvmJsCommonMain/kotlin/com/badoo/reaktive/observable/plugins/ObservablePlugins.kt

```diff
@@ -0,0 +1,11 @@
+package com.badoo.reaktive.observable.plugins
+
+import com.badoo.reaktive.annotations.ExperimentalReaktiveApi
+import kotlin.jvm.JvmField
+
+@ExperimentalReaktiveApi
+actual object ObservablePlugins {
+
+    @JvmField
```

Comment:
Make it also JvmStatic to remove singleton access?

Reply:
Good catch!

Reply:
So @JvmStatic can not be used together with @JvmField. And actually @JvmField in `object` makes it `static field`. So there is no singleton access on read/write.